code/modules/clothing/neck/_neck.dm

```diff
@@ -18,36 +18,125 @@
 			. += mutable_appearance('icons/effects/blood.dmi', "maskblood")
 
 /obj/item/clothing/neck/tie
-	name = "tie"
+	name = "clip-on tie"
 	desc = "A neosilk clip-on tie."
 	icon = 'icons/obj/clothing/neck.dmi'
-	icon_state = "bluetie"
+	icon_state = "detective"
 	inhand_icon_state = "" //no inhands
 	w_class = WEIGHT_CLASS_SMALL
 	custom_price = PAYCHECK_CREW
 
-/obj/item/clothing/neck/tie/blue
+/obj/item/clothing/neck/tie/greyscale
+	greyscale_colors = "#4d4e4e"
+	name = "slick tie"
+	desc = "A neosilk tie."
+	icon_state = "tie_greyscale_tied"
+	greyscale_config = /datum/greyscale_config/ties
+	greyscale_config_worn = /datum/greyscale_config/ties_worn
+	flags_1 = IS_PLAYER_COLORABLE_1
+	/// All ties start untied unless otherwise specified
+	var/is_tied = FALSE
+	/// How long it takes to tie the tie
+	var/tie_timer = 4 SECONDS
+
+/obj/item/clothing/neck/tie/greyscale/Initialize(mapload)
+	. = ..()
+	update_appearance(UPDATE_ICON)
+	if(ismob(loc))
+		var/mob/wearer = loc
+		wearer.update_clothing(ITEM_SLOT_NECK)
```

Comment:
why is this required exactly?

Reply:
If it starts spawned in a player's inventory it needs to update its icon

Reply:
When would it spawn in the player's inventory? shouldn't it be equipped to the relevant slot later?
This pattern worries me

Reply:
outfits like the psychologist's main one come to mind, as well as some others.

Reply:
if this is a problematic thing, I can just set the tied versions of the tie to have the correct icon_state without needing their appearance updated
The reason that update is there is because of the vending machines. On the vending machine, the tie visually displays as tied because its `icon_state` is set to tied, but when it's dispensed, it updates to its proper untied state. If they start off as tied, they actually wouldn't even need this updated icon state, and I realize now as I type this that it's a completely redundant thing.